koji/playbooks/make-modules-metadata.yaml

```
@@ -0,0 +1,24 @@
+---
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: Collect RPM names
+      command: "rpm -qp {{ repository_path }}/*rpm --queryformat=\"%{name}-%{epochnum}:%{version}-%{release}.%{arch}\n\""
```

Isn't this easier? Note that this doesn't include epoch if epoch is 0, but I'd expect the tooling to understand that.
```suggestion
      command: "rpm -qp {{ repository_path }}/*rpm --queryformat='%{nevra}\n'"
```

And in scripts I prefer long options for readability:
```suggestion
      command: "rpm --query --package {{ repository_path }}/*rpm --queryformat='%{nevra}\n'"
```

---

Easier yes, but that will not generate correct information. To quote [the spec](https://github.com/fedora-modularity/libmodulemd/blob/main/yaml_specs/modulemd_stream_v2.yaml#L668):

>     artifacts:

        # rpms:
        # RPM artifacts shipped with this module
        # A set of NEVRAs associated with this module. An epoch number in the
        # NEVRA string is mandatory.

---

Thanks! That would make a good comment to prevent anyone in the future trying to "optimize" it :)